doc_source/tutorials-cloudformation-codecommit.md

```diff
@@ -12,7 +12,7 @@ You must have created the following resources to use with the AWS CloudFormation
   Unzip the file and place it on your local computer\.
 + Download the [SampleApp\_Linux\.zip](samples/SampleApp_Linux.zip) sample application file\.
 
-1. Unzip the files from [SampleApp\_Linux\.zip](samples/SampleApp_Linux.zip) and upload the files into to your AWS CodeCommit repository\. You must upload the unzipped files to the root directory of your repository\. You can follow the instructions in [Step 2: Add Sample Code to Your CodeCommit Repository](tutorials-simple-codecommit.md#codecommit-add-code) to push the files to your repository\.
+1. Unzip the files from [SampleApp\_Linux\.zip](samples/SampleApp_Linux.zip) and upload the files to your AWS CodeCommit repository\. You must upload the unzipped files to the root directory of your repository\. You can follow the instructions in [Step 2: Add Sample Code to Your CodeCommit Repository](tutorials-simple-codecommit.md#codecommit-add-code) to push the files to your repository\.
```

Comment:
The only change is deleting the word, `into`

Reply:
Hello, Thank you so much for your pull request! Thank you for catching this error in the CloudFormation pipeline  tutorial. Merging now and will publish the updated doc here: https://docs.aws.amazon.com/codepipeline/latest/userguide/tutorials-cloudformation-codecommit.html

Appreciate your help with the docs!